src/main/java/org/sefglobal/scholarx/util/EmailUtil.java

```diff
@@ -1,42 +1,33 @@
 package org.sefglobal.scholarx.util;
 
-import org.sefglobal.scholarx.model.Email;
-import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.mail.javamail.MimeMessageHelper;
-import org.springframework.stereotype.Component;
-import org.thymeleaf.context.Context;
+import org.sefglobal.scholarx.model.Mail;
+import org.sefglobal.scholarx.service.MailConnection;
+import org.simplejavamail.api.email.Email;
+import org.simplejavamail.email.EmailBuilder;
 import org.thymeleaf.spring5.SpringTemplateEngine;
+import org.thymeleaf.context.Context;
 
-import javax.mail.MessagingException;
-import javax.mail.internet.MimeMessage;
-import java.nio.charset.StandardCharsets;
-
-
-@Component
 public class EmailUtil {
 
-    private final JavaMailSender emailSender;
-
     private final SpringTemplateEngine templateEngine;
 
-    public EmailUtil(JavaMailSender emailSender, SpringTemplateEngine templateEngine) {
-        this.emailSender = emailSender;
+    public EmailUtil(SpringTemplateEngine templateEngine) {
         this.templateEngine = templateEngine;
     }
 
-    public void sendSimpleMessage(Email email) throws MessagingException {
-        MimeMessage message = emailSender.createMimeMessage();
-        MimeMessageHelper helper = new MimeMessageHelper(message, MimeMessageHelper.MULTIPART_MODE_MIXED_RELATED, StandardCharsets.UTF_8.name());
+    public void sendEmail(Mail mail) {
 
         Context context = new Context();
-        context.setVariables(email.getProps());
+        context.setVariables(mail.getProps());
 
-        String html = templateEngine.process("scholarx", context);
+        String htmlText = templateEngine.process("scholarx", context);
 
-        helper.setTo(email.getEmail());
-        helper.setSubject(email.getSubject());
-        helper.setText(html, true);
+         Email email = EmailBuilder.startingBlank()
+                .to(mail.getEmailAddress())
```

Comment:
Here we will have to include the from() field also. There may be a way to get the `from` field automatically from the MailBuilder, but for now, let's put the senders email manually. Sorry about my previous comment on this

Reply:
So we have to get the sender's email through environment variables right? @Piumal1999

Reply:
Yeah @pasindur99